src/Symfony/Component/Console/Tests/phpt/single_application/help_name.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+Single Application can be executed
+--ARGS--
+--help
+--FILE--
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\SingleCommandApplication;
+
+$vendor = __DIR__;
+while (!file_exists($vendor.'/vendor')) {
+    $vendor = dirname($vendor);
+}
+require $vendor.'/vendor/autoload.php';
+
+(new SingleCommandApplication())
+    ->setName('My Super Command')
+    ->setCode(function (InputInterface $input, OutputInterface $output) {
+    })
+;
+?>
+--EXPECTF--
+Usage:
+  Standard input code
```

Comment:
What's that?

Reply:
This is weird. I guess it's because if the phphunit wrapper. It's supposed to be the PHP filename:

Usage:
    test.php arg1 arg2

Reply:
the phpunit wrapper has nothing to do here
php 7.4 changed the output, that's it

Reply:
I did not understand. 

I'm not talking about the bridge but about how phphunit run phpt test. In local I use php 7.4 and I don't have such issue when I use 'php test.php --help'

Reply:
oh ok sorry :)
so on PHP < 7.4 this might be a dash (-) instead, isn't it?

Reply:
Tests are green on PHP 7.3 too (on travis and on my computer) So I guess it's here for years.

Anyway, in "real life", it works well on 7.2+:
```
>…oire/dev/github.com/lyrixx/symfony(console-single-command) php7.2 test.php  --help
Usage:
  test.php

Options:
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
>…oire/dev/github.com/lyrixx/symfony(console-single-command) php7.3 test.php  --help
Usage:
  test.php

Options:
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
>…oire/dev/github.com/lyrixx/symfony(console-single-command) php7.4 test.php  --help
Usage:
  test.php

Options:
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
```